openshift/template.yaml

```diff
@@ -0,0 +1,173 @@
+apiVersion: v1
+kind: Template
+labels:
+  template: f8a-analytics-ingestion
+metadata:
+  name: f8a-analytics-ingestion
+  annotations:
+    description: f8a-analytics-ingestion
+objects:
+- apiVersion: v1
+  kind: DeploymentConfig
+  metadata:
+    labels:
+      service: f8a-analytics-ingestion
+    name: f8a-analytics-ingestion
+  spec:
+    replicas: "${{REPLICAS}}"
+    selector:
+      service: f8a-analytics-ingestion
+    strategy:
+      type: Recreate
+      recreateParams:
+        pre:
+          failurePolicy: Retry
+          execNewPod:
+            containerName: f8a-analytics-ingestion
+            command: ["ingestion-pre-hook.sh"]
+    template:
+      metadata:
+        labels:
+          service: f8a-analytics-ingestion
+      spec:
+        containers:
+        - command:
+          - /bin/entrypoint.sh
+          env:
```

Comment:
Who/where/when will set the `INGESTION_RUN_DB_MIGRATIONS` env. variable? This variable is required by `run-db-migrations.sh` script.